node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@scalar-labs/scalar-admin",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
we update this because now the version of Java and Node are aligned so they have to be updated at the same time.